src/btree/bt_debug.c

```diff
@@ -204,8 +204,8 @@ __wt_debug_addr_print(
 	WT_DECL_ITEM(buf);
 
 	WT_RET(__wt_scr_alloc(session, 128, &buf));
-	fprintf(stderr, "%s\n",
-	    __wt_addr_string(session, addr, addr_size, buf));
+	WT_RET(__wt_fprintf(stderr, "%s\n",
+	    __wt_addr_string(session, addr, addr_size, buf)));
 	__wt_scr_free(session, &buf);
```

Comment:
@keithbostic, we probably want to make sure the `buf` scratch buffer is freed if there's an error from `__wt_fprintf`.